draft-irtf-panrg-path-properties.md

```diff
@@ -140,6 +140,8 @@ For example, a path property based on an approximate calculation may describe th
 The terminology defined in this document is intended to be general and applicable to existing and future path-aware technologies.
 Using this terminology, a path-aware technology can define and consider specific path elements and path properties on a specific level of abstraction.
 For instance, a technology may define path elements as IP routers, e.g., in source routing ({{?RFC1940}}). Alternatively, it may consider path elements on a different layer of the Internet Architecture ({{?RFC1122}}) or as a collection of entities not tied to a specific layer, such as an AS or an ERO.
+Even within a single path-aware technology, terminology might differ depending on the context in which it is used.
```

Comment:
Is it really the terminology that differs, or is it the specific definition you get when using this terminology, i.e., what specific component gets defined as a node/path/endpoint? (At least that's what the prior few sentences suggest, as I read them.)

I'm wondering if this should say:
```suggestion
Even within a single path-aware technology, specific definitions might differ depending on the context in which they are used.
```